src/main/java/seedu/address/logic/commands/FindByTagsCommand.java

```diff
@@ -0,0 +1,15 @@
+package seedu.address.logic.commands;
+
+import seedu.address.model.person.TagsContainKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose tags contains any of the argument keywords.
+ * Keyword matching is case sensitive.
+ */
+public class FindByTagsCommand extends FindCommand {
+    public static final String COMMAND_OPTION = "tag";
```

Comment:
Should this have `-` in front i.e. `-tag`?

Reply:
I'm handling the prefix in OptionBearingArgument instead of locally in the FindByTagsCommand. This is to abstract handling of options to another class to hopefully keep our command designs simpler

Reply:
Nice! 👍 Re-read :) I could use your `OptionBearingArgument` class too 👍